### DISCUSSION TITLE
Is it possible to set the currency in the GitHub App?

### DISCUSSION THREAD

**Discussion by endakelly** [infracost/infracost#3085] [Q&A]
The github app does all I need but the currency is set to USD and I cannot see a way to change that like can be done with the CLI. Has anyone been able to do this?

**Comment by vdmgolub** [comment]
Hello! You can set [these env vars](https://www.infracost.io/docs/features/environment_variables/#infracost_currency) for your current GitHub integration. For example:
1. Go to Infracost Cloud. Navigate to Org Settings -> Integrations section -> pick your integration.
2. Click "Next" to go to settings
3. Click "Yes" in "Do you need additional environment variables?"
4. Add the env var `INFRACOST_CURRENCY`, for example, with `EUR` code:
   <img width="976" alt="image" src="https://github.com/infracost/infracost/assets/87082/e4266d73-db07-439c-b857-14da228fd1ad">
5. Click "Complete setup".

The next estimate should have the new currency.

You can also specify the format, see the docs I've mentioned above.

Hope this helps :)

**Reply by endakelly** [reply]
Ah legend, I completely missed the environment variables part of the integration settings. Working a treat now
